__tests__/app.test.js

```diff
@@ -40,7 +40,7 @@ describe("1. GET /api/topics", () => {
   });
 });
 describe("GET /api/articles/:article_id", () => {
-  test("status:200, responds with single matching article object", () => {
+  test.only("status:200, responds with single matching article object", () => {
```

Comment:
Be careful of commiting these `.onlys` it's an easy way to skip important tests and introduce bugs by accident.

Reply:
I have it removed now Thanks for this, Paul! I need to remember about these little things :D 
Is I OK to merge it now to the main